include/picongpu/particles/particleToGrid/ComputeGridValuePerFrame.hpp

```diff
@@ -54,9 +54,9 @@ ComputeGridValuePerFrame<T_ParticleShape, T_DerivedAttribute>::getUnitDimension(
 
 template<class T_ParticleShape, class T_DerivedAttribute>
 HINLINE std::string
-ComputeGridValuePerFrame<T_ParticleShape, T_DerivedAttribute>::getName() const
+ComputeGridValuePerFrame<T_ParticleShape, T_DerivedAttribute>::getName()
```

Comment:
missing spaces inside  < >

Reply:
not a new file but a minimal patch for an existing, legacy file.
Pls let us focus on the content and only mark style for new code.